flocker/common/script.py

```diff
@@ -23,10 +23,10 @@
 from .. import __version__
 
 try:
-    from ._journald import sd_journal_send
-except OSError as e:
-    # This platform doens't have journald.
-    sd_journal_send = None
+    from eliot.journald import JournaldDestination
```

Comment:
I wonder why it's not `eliot.JournaldDestination` to match with, eg, `eliot.FileDestination`...